src/js/entry.js

```diff
@@ -208,3 +209,32 @@ export function rootApp() {
         );
     }
 }
+
+export function noAccess() {
+    const { store } = spinUpStore();
+    window.insights.chrome.auth.getUser().then(({ entitlements }) => {
```

Comment:
We should catch this if it fails

Reply:
@ryelo what would be a good way to catch when it fails here? throw some logs?

Reply:
Throw a log or an error of some kind. This will generate sentry errors if we don't catch it!